taca/backup/backup.py

```diff
@@ -257,39 +276,63 @@ def _clean_tmp_files(self, files):
 
     def _log_pdc_statusdb(self, run):
         """Log the time stamp in statusDB if a file is succussfully sent to PDC."""
-        try:
-            run_vals = run.split("_")
-            if len(run_vals[0]) == 8:
-                run_date = run_vals[0][2:]
-            else:
-                run_date = run_vals[0]
-            run_fc = f"{run_date}_{run_vals[-1]}"
-            couch_connection = statusdb.StatusdbSession(self.couch_info).connection
-            db = couch_connection[self.couch_info["db"]]
-            fc_names = {e.key: e.id for e in db.view("names/name", reduce=False)}
-            d_id = fc_names[run_fc]
-            doc = db.get(d_id)
-            doc["pdc_archived"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-            db.save(doc)
-            logger.info(
-                f'Logged "pdc_archived" timestamp for fc {run} in statusdb doc "{d_id}"'
-            )
-        except:
-            logger.warn(f'Not able to log "pdc_archived" timestamp for run {run}')
+        if "_AV" not in run:
```

Comment:
Will we need ONT here too?

Reply:
Yes, if we want the timestamp for ONT (probably, but we can add that later). At the moment it just skips ONT.